Library/Homebrew/keg.rb

```diff
@@ -339,14 +339,14 @@ def to_formula
   end
 
   def installed_dependents
+    return [] unless linked?
```

Comment:
This will presumably return for anything `keg_only`? I'm not sure that's what we want. Perhaps we should still do it for things that aren't linked, too, but just go purely by `name` (which I realise contradicts what I said earlier).